hypha/apply/funds/blocks.py

```diff
@@ -38,7 +38,7 @@ class Meta:
 class ValueBlock(ApplicationSingleIncludeFieldBlock):
     name = 'value'
     description = 'The value of the project'
-    widget = forms.NumberInput(attrs={'min': 0})
+    widget = forms.NumberInput(attrs={'min': 0, 'id':'requested-amount'})
```

Comment:
Instead of going the js route. I would advice to override the NumberInput template and use that instead. 

Even if we js route, which to me feel little hackish, you should use `class` attribute instead of the `id` to allow for multiple valueblock to exist on the same page.

Reply:
I concur, the js script feels like a hack. This should be possible to fix directly in the template for the widget.

Reply:
`ValueBlock` inherits `ApplicationSingleIncludeFieldBlock`, hence it can only exist once in a form.